.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
   # pytype needs py-2.7, mypy needs py-3.3+. Additional logic in runtests.py
   - if [[ $TRAVIS_PYTHON_VERSION == '3.6-dev' ]]; then pip install -U flake8==3.3.0 flake8-bugbear>=17.3.0 flake8-pyi>=17.1.0; fi
   - if [[ $TRAVIS_PYTHON_VERSION == '3.5' ]]; then pip install -U git+git://github.com/python/mypy git+git://github.com/python/typed_ast; fi
-  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then pip install -U git+git://github.com/google/pytype; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then pip install -U git+git://github.com/google/pytype; apt-get install python3; fi
```

Comment:
I don't think this is needed. Python 3 should be installed on the image even if the environment is Python 2. You should be able to invoke the system Python via `python3`.

If not, this command would need sudo to lock the proper files.

Reply:
thanks, will experiment with that too. the issue might be that the system python3 is not symlinked as python3.6 in bin (pytype expects to invoke python with a specific minor version; i'm having a hard time getting travis to support that)

Reply:
ouch, looks like we need system python to be python2.7 too. we'll need to create a python3.6 symlink, i think.

Reply:
Since the image will have 3.6 somewhere, you might also be able to make a virtualenv and specify the python version.

Reply:
the problem is i would need python2.7 and python3.6 installed in the same virtualenv, since pytype depends on python2.7 to build and run, but needs to shell out to the python version of the code it's analysing at runtime.